src/ZoneServer/World/Actors/Monsters/Mob.cs

```diff
@@ -290,8 +290,21 @@ public void Kill(ICombatEntity killer)
 			this.Properties.SetFloat(PropertyName.HP, 0);
 			this.Components.Get<MovementComponent>()?.Stop();
 
-			var expRate = ZoneServer.Instance.Conf.World.ExpRate / 100.0;
-			var classExpRate = ZoneServer.Instance.Conf.World.ClassExpRate / 100.0;
+			var worldConf = ZoneServer.Instance.Conf.World;
+
+			var expRate = worldConf.ExpRate / 100.0;
+			var classExpRate = worldConf.ClassExpRate / 100.0;
+
+			if (this.IsBuffActive(BuffId.SuperExp))
+			{
+				expRate *= worldConf.BlueJackpotExpRate;
```

Comment:
The default value of this option is 10000... maybe my math is off, but doesn't this mean you're increasing the EXP rate by 10,000,000%? That seems rather high, doesn't it?

Reply:
Oops, that's an error, it should be dividing that by 100F.